packages/bitcore-client/bin/wallet-derive

```diff
@@ -28,8 +28,12 @@ const main = async () => {
     let index = wallet.addressIndex || 0;
 
     while (unusedAddressCounter < gap) {
-      const address = await wallet.nextAddressPair(change);
-
+      let address;
+      if (wallet.lite) {
+        address = await wallet.nextAddressPairLite(change);
```

Comment:
I think if you made nextAddressPair do this switch, then you wouldn't have to change this part.